src/renderer/components/InfoDropdown.tsx

```diff
@@ -13,9 +13,84 @@ import { FaAngleDown } from 'react-icons/fa';
 import Accordion from 'react-bootstrap/Accordion';
 import Card from 'react-bootstrap/Card';
 
-type Props = {};
+// import { Service } from '../App.d';
+
+type Props = {
+  service?: any;
```

Comment:
Shouldn't this have type: Service

Reply:
doesn't work with Service

Reply:
i think you can help with resolving the type maybe haha